packages/sitecore-jss-angular-schematics/tsconfig-esm.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "es6",
+    "outDir": "dist/esm"
+  },
+}
```

Comment:
Looks like we need a newline here and in all other `tsconfig-esm.json` files.

Reply:
Good catch, fixed.

Reply:
Strange that I added newlines to each file and committed them, but the newline doesn't show up on GitHub... I see them in my editor though.

Reply:
Hmm, I'm seeing them as fixed too. Maybe a caching thing?